src/ixdat/techniques/deconvolution.py

```diff
@@ -0,0 +1,241 @@
+"""Module for deconvolution of mass transport effects."""
```

Comment:
So, this module in general abbreviates more than I prefer. I will just leave this one comment on it.